kuma/users/templatetags/jinja_helpers.py

```diff
@@ -12,15 +12,22 @@
 from kuma.core.templatetags.jinja_helpers import datetimeformat
 from kuma.core.urlresolvers import reverse
 
-from ..jobs import UserGravatarURLJob
-
 
 @library.global_function
-def gravatar_url(email, secure=True, size=220, rating='pg',
-                 default=settings.DEFAULT_AVATAR):
-    job = UserGravatarURLJob()
-    return job.get(email, secure=secure, size=size,
-                   rating=rating, default=default)
+def get_avatar_url(user, provider='github'):
+    """
+    Get the user's avatar URL for the specified provider. Returns None if the
+    user is not logged-in, or the default avatar if the user is logged-in but
+    has no avatar for the specified provider.
+    """
+    if user and user.is_authenticated:
+        for account in user.socialaccount_set.filter(provider=provider):
```

Comment:
Some day, when you can log in with either (github or google), there'd need to be a smart way to merge accounts. That day, how do you know which socialaccount to use? Perhaps whichever one you most recently logged in with??

Reply:
Good question. I guess we'll cross that bridge when we get there.